mindsdb/integrations/handlers/notion_handler/README.md

```diff
@@ -0,0 +1,100 @@
+# Notion Handler
+
+This is the implementation of the Notion handler for MindsDB.
+
+## Notion
+Notion is a note-taking and productivity freemium cloud platform.
+In short, notion has all-in-one workspace tool that integrates kanban boards, tasks, wikis, and database.
+In this handler. python client of api is used and more information about this python client can be found [here](https://pypi.org/project/notion-client/)
```

Comment:
Let's remove this line and add the following link,
https://www.notion.so/help/guides/what-is-notion